cpp/include/kvikio/remote_handle.hpp

```diff
@@ -180,11 +185,12 @@ class S3Endpoint : public RemoteEndpoint {
              std::optional<std::string> aws_region            = std::nullopt,
              std::optional<std::string> aws_access_key        = std::nullopt,
              std::optional<std::string> aws_secret_access_key = std::nullopt,
+             std::optional<std::string> aws_session_token     = std::nullopt,
```

Comment:
Can we add the new parameter to the end of the argument list, to avoid an API break? The types of `aws_session_token` and `aws_endpoint_url` are identical, so users of the old API will now be passing an endpoint as if it was a session token.

Reply:
I've made the change. It would be desirable to keep them grouped together but a hidden breaking API change sounds troublesome.

Reply:
Turns out I only changed the order in the .cpp file, should be fixed now! Also added missing documentation for the new parameter to that function.